climada_petals/hazard/test/data/cxml_sample_transformation.xsl

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
isn't this file the same as tc_tracks_forecast_cxml2csv.xsl ?

Reply:
Closing this one as well.